sycl/test/spec_const/spec_const_hw.cpp

```diff
@@ -5,7 +5,8 @@
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
 // TODO: re-enable after CI drivers are updated to newer which support spec
 // constants:
-// XFAIL: acc,cpu,cuda,gen
+// XFAIL: linux
+// UNSUPPORTED: cuda
```

Comment:
FYI. Post-commit testing is failing due to what “expected to fail” condition.
There are a few ways how to fix it, but I guess the easiest one is to replace:
```c++
// UNSUPPORTED: cuda
```
with 
```c++
// REQUIRES: opencl
```